gradio/components/code.py

```diff
@@ -163,3 +163,8 @@ def example_payload(self) -> Any:
 
     def example_value(self) -> Any:
         return "print('Hello World')"
+
+    def process_example(self, value: str | tuple[str] | None) -> str | None:
+        if isinstance(value, tuple):
+            return value[0]
```

Comment:
I would suggest only showing the name of the file, not the whole path, to be consistent with: https://github.com/gradio-app/gradio/blob/c62a57e7f8f2f6dad0110d06e915c48e7f628073/gradio/components/audio.py#L307